tox.ini

```diff
@@ -3,7 +3,7 @@ envlist = py26, py27, flake8
 
 [testenv]
 deps=
-  pytest
+  pytest<2.8.0
```

Comment:
Yeah, it was the import of `call` from Mock.  See https://github.com/pytest-dev/pytest/issues/1044 (which is a dup of another one).